IdentityCore/src/oauth2/aad_base/MSIDAADOauth2Factory.m

```diff
@@ -111,6 +111,7 @@ - (BOOL)verifyResponse:(MSIDAADTokenResponse *)response
             {
                 errorCode = MSIDErrorServerProtectionPoliciesRequired;
                 additionalUserInfo[MSIDUserDisplayableIdkey] = response.additionalUserId ?: @"";
```

Comment:
TODO: do we need empty string here? Seems we don't need this unless it is a public protocol agreement.

Reply:
Agreed to set nil here.